### ISSUE TITLE
Error on building larlitecv tmw_muon_tagger branch

### ISSUE THREAD

**Issue by vgenty** [LArbys/larlitecv#24]
Am I on the right branch? I get this error when building.

```Compiling TaggerContourTools...
make[1]: Entering directory `/usr/local/share/dllee_unified/larlitecv/app/TaggerContourTools'
<< compiling ContourGapFill.cxx >>
ContourGapFill.cxx: In member function 'int larlitecv::ContourGapFill::connectSpan(const larlitecv::ContourGapFill::BadChSpan&, const std::vector<larlitecv::ContourShapeMeta>&, const larcv::Image2D&, const larcv::Image2D&, larcv::Image2D&)':
ContourGapFill.cxx:350:26: error: 'const class larlitecv::ContourShapeMeta' has no member named 'getPCAStartdir'
      leftdir.x = ctrleft.getPCAStartdir()[0];
                          ^
ContourGapFill.cxx:351:22: error: 'const class larlitecv::ContourShapeMeta' has no member named 'getPCAStartdir'
  leftdir.y = ctrleft.getPCAStartdir()[1];
                      ^
ContourGapFill.cxx:356:26: error: 'const class larlitecv::ContourShapeMeta' has no member named 'getPCAEnddir'
      leftdir.x = ctrleft.getPCAEnddir()[0];
                          ^
ContourGapFill.cxx:357:26: error: 'const class larlitecv::ContourShapeMeta' has no member named 'getPCAEnddir'
      leftdir.y = ctrleft.getPCAEnddir()[1];
                          ^
ContourGapFill.cxx:397:30: error: 'const class larlitecv::ContourShapeMeta' has no member named 'getPCAStartdir'
        rightdir.x = ctrright.getPCAStartdir()[0];
                              ^
ContourGapFill.cxx:398:26: error: 'const class larlitecv::ContourShapeMeta' has no member named 'getPCAStartdir'
    rightdir.y = ctrright.getPCAStartdir()[1];
                          ^
ContourGapFill.cxx:403:30: error: 'const class larlitecv::ContourShapeMeta' has no member named 'getPCAEnddir'
        rightdir.x = ctrright.getPCAEnddir()[0];
                              ^
ContourGapFill.cxx:404:30: error: 'const class larlitecv::ContourShapeMeta' has no member named 'getPCAEnddir'
        rightdir.y = ctrright.getPCAEnddir()[1];
                              ^
make[1]: *** [/usr/local/share/dllee_unified/larlitecv/build/TaggerContourTools/ContourGapFill.o] Error 1
make[1]: Leaving directory `/usr/local/share/dllee_unified/larlitecv/app/TaggerContourTools'
make: *** [obj] Error 2
Singularity singularity-dllee-unified-11272017.img:/usr/local/share/dllee_unified/larlitecv> 
```

**Comment by vgenty** [comment]
If anyone else is stuck, the usual workaround is to disable this module in the top level makefile

**Comment by twongjirad** [comment]
I forgot to commit the relevant header (TaggerContourTools/ContourShapeMeta.h).  This is fixed as of de7e66b58c59dd8ef7bfb2f1f4eebb8fcf296325.
